File: src/main/java/src/gestorAplicacion/Calendario/TareaEstudiante.java
package gestorAplicacion.Calendario;

import gestorAplicacion.personas.Estudiante;
import java.io.Serializable;

public class TareaEstudiante implements Serializable{
    private Tarea tarea;
    private Estudiante estudiante;
    private double grade;

    public TareaEstudiante(Tarea tarea, Estudiante estudiante, double grade) {
        this.tarea=tarea;
        this.estudiante=estudiante;
        this.grade=grade;
    }

	public Estudiante getEstudiante() {
		return estudiante;
	}

	public void setEstudiante(Estudiante estudiante) {
		this.estudiante = estudiante;
	}

	public double getGrade() {
		return grade;
	}

	public void setGrade(double grade) {
		this.grade = grade;
	}

	public Tarea getTarea() {
		return tarea;
	}

	public void setTarea(Tarea tarea) {
		this.tarea = tarea;
	}

    
}

File: src_py/Calendario/TareaEstudiante.py


class TareaEstudiante:
    def __init__(self, tarea, estudiante, grade: float):
        self.__tarea = tarea
        self.__estudiante = estudiante
        self.__grade = grade

    def getEstudiante(self):
        return self.__estudiante

    def setEstudiante(self, estudiante):
        self.__estudiante = estudiante

    def getGrade(self) -> float:
        return self.__grade

    def setGrade(self, grade: float):
        self.__grade = grade

    def getTarea(self):
        return self.__tarea

    def setTarea(self, tarea):
        self.__tarea = tarea

    def __str__(self) -> str:
        return f'TareaEstudiante{{tarea={self.__tarea}, estudiante={self.__estudiante}, grade={self.__grade}}}'
